FirebaseRemoteConfig/Sources/RCNConfigRealtime.m

```diff
@@ -585,7 +602,9 @@ - (void)URLSession:(NSURLSession *)session
                  }];
       FIRLogError(kFIRLoggerRemoteConfig, @"I-RCN000021", @"Cannot establish connection. Error: %@",
                   error);
-      [self propogateErrors:error];
+      if (statusCode != 403) {
```

Comment:
is there no other info in the response that indicates what kind of error it is? feels fairly brittle to be checking for a literal string.. 
i might prefer just returning the full message for all 403s, since that will likely require user intervention to be fixed

Reply:
The status code and response body are processed in separate delegate methods so we can't just return all 403s. However all server errors have ' code: <HTTP_CODE> ' in the response so I think we can match on the string 
' code:403 ' instead. Updated.